Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1054,6 +1056,24 @@ inline ScrollSnapStop BuilderConverter::convertScrollSnapStop(BuilderState&, con
     return fromCSSValue<ScrollSnapStop>(value);
 }
 
+inline std::optional<ScrollbarColor> BuilderConverter::convertScrollbarColor(BuilderState& builderState, const CSSValue& value)
+{
+    if (is<CSSPrimitiveValue>(value)) {
+        ASSERT(value.valueID() == CSSValueAuto);
+        return std::nullopt;
+    }
+
+    ASSERT(value.isPair());
+    const auto& pair = downcast<CSSValuePair>(value);
+
+    ScrollbarColor result = {
+        builderState.colorFromPrimitiveValue(downcast<CSSPrimitiveValue>(pair.first())),
+        builderState.colorFromPrimitiveValue(downcast<CSSPrimitiveValue>(pair.second()))
+    };
+
+    return std::make_optional<ScrollbarColor>(result);
```

Comment:
This will copy the colors, which is wasteful if they are complex colors. Easy to avoid. Here is one way.
```
return WTFMove(result);
```
Another way:
```
return ScrollbarColor {
    builderState.colorFromPrimitiveValue(downcast<CSSPrimitiveValue>(pair.first())),
    builderState.colorFromPrimitiveValue(downcast<CSSPrimitiveValue>(pair.second()))
};
```

Reply:
Okay thanks!